README.md

```diff
@@ -1,36 +1,59 @@
 # System Test
 
 ### How to build the testing docker image
-`docker build --platform linux/amd64 --no-cache -t stellar/stellar-system-test -f Dockerfile .`
+`docker build --platform linux/amd64 --no-cache -t stellar/system-test -f Dockerfile .`
 
 ### Run tests from the docker image:
 In short term, running tests on the `stellar/stellar-system-test` docker image is only supported on hosts that are on x86/amd cpu platforms. Arm cpu platforms are not supported for docker usage yet, this includes any Apple M1 device. If you are in the latter group, then can still run the tests but will need to refer to running tests directly from checked out repo instead.
 
-Running docker image, the following command line args can be passed, example values used, change to your settings:
+Running docker image, this is an example of command line args, please change these to settings that make sense for you expectations:
 ```
-docker run --platform linux/amd64 --rm -it --name e2e_test stellar/stellar-system-test:latest \
+docker run --platform linux/amd64 --rm -it --name e2e_test stellar/system-test:latest \
 --SorobanCLICrateVersion 0.2.1 \
---SorobanExamplesGitHash main \
---SorobanExamplesRepoURL "https://github.com/stellar/soroban-examples.git" \
---RustToolchainVersion "stable" \
---CoreDebianVersion "19.5.1-1111.eba1d3de9.focal~soroban" \
---HorizonDebianVersion "2.22.0~soroban-304" \
---SorobanRPCDebianVersion "0.0.1~alpha-2" \
---TestFilter "<filter>" \
+--CoreDebianVersion "19.5.1-1137.b3a6bc281.focal~soroban" \
```

Comment:
I think that you'll need to update this one, right ?

Reply:
it's example values to demonstrate version usage in docs, wasn't intending to keep these current or I replace all of these with substitution blocks and no real versions?

Reply:
Ahh. I missed that it was a Readme file. I'm not picky about this one than.